docs/src/Link.doc.js

```diff
@@ -111,4 +138,68 @@ card(
   />
 );
 
+card(
+  <Example
+    id="advancedExample"
+    name="Advanced Example"
+    defaultCode={`
+function AdvancedExample() {
+  const [preventDefault, setPreventDefault] = React.useState(true);
+  const onClick = ({ event }) => {
+    if (preventDefault) {
+      event.preventDefault();
+    }
+  };
+
+  return (
+    <Box>
+      <Row gap={2} padding={1}>
+        <Box paddingX={2}>
```

Comment:
@jackhsu978 now that we use `<Row gap={2} />`, we no longer need this `<Box paddingX={2}>`.

I would change it to the following:
```
      <Row gap={2} padding={2}>
        <Label htmlFor="preventDefault">
          <Text>Prevent default on tap</Text>
        </Label>
        <Switch
          id="preventDefault"
          onChange={() => setPreventDefault(!preventDefault)}
          switched={preventDefault}
        />
      </Row>
      <Divider />
```

To fix alignment + make it obvious that the toggle changes the examples below.

Reply:
done. thank you for suggestions!